rtree: strip final .git from clone URLs

SuperSandro2000 opened the pull request:
otherwise paths contain a final .git when using the clone URLs copied from the GitHub Web UI

majewsky reviewed (changes requested):
I don't see how this patch solves the issue that the title describes.

SuperSandro2000 committed:
rtree: strip final .git from clone URLs

otherwise paths contain a final .git when using the clone URLs copied from the GitHub Web UI

majewsky commented:
I added the other half of the fix in 8cea64eb2a4e51275bda51533536f9d7ad358208 and merged.
